src/common/components/Toast/utils/openToast.ts

```diff
@@ -0,0 +1,18 @@
+import { Bounce, toast } from "react-toastify";
+import { ToastText } from "@/common/components/Toast/constants/ToastText";
+
+type TostType = keyof ToastText;
+
+export default function openToast(type: TostType, text: string, time: number = 3000) {
```

Comment:
기본값을 줄때는 보통 타입 지정을 하지 않는 편입니다~
```suggestion
export default function openToast(type: TostType, text: string, time = 3000) {
```